topics/imaging/tutorials/yolo_prediction/tutorial.md

```diff
@@ -0,0 +1,239 @@
+Vers polynoidés
+```
+
+Each class name must be on its own line, in the same order used during model training. So the class ID 0 corresponds to Buccinidae, and 1 to Bythograeidae.
+
+- `Model`: Upload and choose from the dataset either `YOLOv8-weights-for-Buccinidae-detection.pt` or `YOLOv8-weights-for-Bythograeidae-detection.pt`, or test both on different runs.
+
+- `Prediction mode`: Select detect. This tells YOLO to output bounding boxes around detected objects.
+
+- `Image size`: Use 1000 (or a smaller number like 640 if processing speed is important). This controls how much the image is resized before prediction. Smaller values = faster but possibly less accurate.
+
+- `Confidence threshold`: Set to 0.25 (25%). This controls how confident the model must be to report a detection. If you increase this value (e.g., 0.5), you’ll get fewer detections, but they’ll be more confident. If you lower it (e.g., 0.1), you may get more results, but possibly more false positives.
+
+- `IoU threshold`: Set to 0.45. This is used for Non-Maximum Suppression (NMS), which removes overlapping detections. A higher IoU value (e.g., 0.7) keeps more overlapping boxes. A lower IoU (e.g., 0.3) removes more overlaps, which may help clean up crowded images.
+
+- `Max detections`: Set a reasonable cap like 300. This limits the number of objects detected per image.
+
+💡 Tip: Try changing the confidence and IoU thresholds to see how detection results vary. It helps you find a good balance between sensitivity and accuracy.
+
+> 💡 **Note**: These models are trained only for detection, not segmentation.
+
+## 🧾 Explore the Outputs
+
+After running the tool, Galaxy will give you several output files for each image. Let’s go through what each one means and how to use them:
+
+📄 **Text files (*.txt):**
+
+These are plain text files containing the detection results. Each line in a file shows:
+
+```
+<class_id> <confidence_score> <x_center> <y_center> <width> <height>
+```
+
+For example: ```0 0.82 350 200 100 120```
+which means:
+
+ - Class ID 0 (in our case, Buccinidae)
+
+ - Detected with 82% confidence
+
+ - The bounding box is centered at (350, 200) and has a width of 100 and height of 120 (in pixels, relative to the image. You can use this file to do further analysis, like counting species or tracking locations over time.
+
+🖼️ **Overlay images (*.jpg):**
+
+These are your original images with colored boxes drawn around detected species. Each box also includes:
+The class name and the confidence score. For example, you might see a box labeled:
+Crabe bythograeid 0.54
+
+![buccinid](../../images/yolo/buccinid.jpeg)
+![bythog](../../images/yolo/bythog2.jpeg)
+
+
+
+These images are useful for visually checking whether detections are correct or if something was missed.
+
+⚠️ No masks or segmentation files
+Since we used detect mode, this tool will not generate segmentation masks (like .tiff or polygon files). Those are only available in segment mode, which we'll cover next.
```

Comment:
The detections are not obtained as a file of coordinates or anything? I mean, it is not possible to use the detections for subsequent analyses? If so, what is the actual use case here?

Reply:
Normally it will. We normaly have a text files with resulting annotation results. "Just" it seems it doesn't work on my test...  BUT, normally we have such text file so for example, we can use it in a next step to sum all species organims detected